app/javascript/packs/containers/NavigationBar/NavigationBarContainer.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import {loadInitialState} from '../../actions/initialState';
+import {getBuildings, getNavBarBuildings} from '../../selectors/buildingsSelector';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import DropdownMenuContainer from './DropdownMenuContainer'
+import {getName, getEmail} from '../../selectors/usersSelector'
+import {getCurrentCategory, getCategories} from "../../selectors/categoriesSelector";
+import CategoryContainer from './CategoryContainer';
+import {getQuestionsByBuildingNavBar, getQuestionsByCategory} from "../../selectors/questionsSelector";
+import {getRemainingAnswersforCategory} from "../../selectors/answersSelector";
+
+
+class NavigationBarContainer extends React.Component {
+    componentDidMount() {
+        if (window.INITIAL_STATE) {
+            this.props.initActions.loadInitialState(window.INITIAL_STATE);
+        }
+    }
+
+    render() {
+        const username = this.props.username;
+        const userEmail = this.props.userEmail;
+        return (
+            <div>
+                <DropdownMenuContainer buildings={this.props.buildings} currentBuilding={this.props.currentBuilding}
```

Comment:
Same comment as above for separating your props for readability. [https://github.com/airbnb/javascript/tree/master/react#props](https://github.com/airbnb/javascript/tree/master/react#props)